packages/thyra/adapters/epetra/src/Thyra_EpetraThyraWrappers.cpp

```diff
@@ -497,6 +497,32 @@ Thyra::get_Epetra_Map(const VectorSpaceBase<double>& vs_in,
 
 }
 
+// Almost like the above one, but working on an RCP vs as input, we can check for the
+// presence of RCP<const Epetra_Map> in the RCP extra data, to save us time.
+Teuchos::RCP<const Epetra_Map>
+Thyra::get_Epetra_Map(const RCP<const VectorSpaceBase<double>>& vs,
+               const RCP<const Epetra_Comm>& comm)
+{
+  //
+  // First, try to grab the Epetra_Vector straight out of the
```

Comment:
Ah, dang! Too much copy and paste. I can fix it. Also, I can amend and force-push to the branch if you want to avoid a trivial commit in the history.

Reply:
@bartgol, yes, please amend and force push.  Either way it will kick off a new round of PR builds (which have had robustness issues lately as described in #3276).  Thank!